src/Dfc.CourseDirectory.WebV2/Features/DataManagement/DataManagementOptions.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace Dfc.CourseDirectory.WebV2.Features.DataManagement
+{
+    public class DataManagementOptions
+    {
+        public TimeSpan ProcessedImmediatelyThreshold { get; set; } = TimeSpan.FromSeconds(5);
```

Comment:
5 seconds feels like a reasonable time here but we can adjust when we having a work end-to-end journey.